wcs-importer-exporter.php

```diff
@@ -50,6 +50,9 @@ public static function init() {
 	 */
 	public static function setup_importer() {
 
+		if ( class_exists( 'WC_Subscriptions' ) && version_compare( WC_Subscriptions::$version, '2.0', '>=' ) ) {
+			add_filter('woocommerce_subscription_get_' . 'last_payment' . '_date', array(WCS_Importer_Exporter::class, 'last_payment_calculation'), 10, 3);
```

Comment:
doh, this was used elsewhere in the class so it's a silly mistake, am too use to using non static classes for adding to filters with class